backend/app/api/v2/endpoints/jobs.py

```diff
@@ -0,0 +1,29 @@
+import json
+from fastapi import Response
+from datetime import datetime, timedelta
+from fastapi import APIRouter
+from ..commons.common import getData
+
+router = APIRouter()
+
+
+@router.get('/api/v2/jobs')
+async def jobsv2(startDate: datetime = None, endDate: datetime = None, pretty: bool = False):
+    if startDate is None:
+        startDate = datetime.utcnow()
+        startDate = startDate - timedelta(days=5)
+
+    if endDate is None:
+        endDate = datetime.utcnow()
+
+    if startDate > endDate:
+        return Response(content="", status_code=422)
```

Comment:
Can we send content as:  invalid date format, start_date must be less than end_date

Reply:
done